src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -301,6 +305,24 @@ class Preview extends React.Component<Props> {
                                 options={this.availableDeviceOptions}
                                 value={this.selectedDeviceOption}
                             />
+                            <Toolbar.Popover
+                                icon="su-calendar"
+                                label={(this.previewStore?.dateTime || new Date()).toLocaleString()}
+                            >
+                                {() => (
+                                    <div className={previewStyles.dateTimeForm}>
+                                        <Form skin="dark">
+                                            <Form.Field label={translate('sulu_admin.preview_date_time')}>
+                                                <DatePicker
```

Comment:
would be nice to add something like a small `x` icon for clearing the field and reset the time to new. but not sure if thats easly possible